src/app/visitor/account/tokens.html

```diff
@@ -0,0 +1,241 @@
+<div class="container">
+    <div class="page-header">
+        <h1>Credit cards</h1>
+    </div>
+
+    <div class="row">
+        <!-- left-side menu -->
+        <ng-include src="'/views/visitor/account/sidebar.html'"></ng-include>
+
+        <!-- tokens info -->
+        <div class="col-xs-12 col-sm-9">
+
+            <!-- note: this is also inside the modal -->
+            <!-- error message -->
+            <ot-message-manager item="message"></ot-message-manager>
+
+            <div class="clearfix"></div>
+            <!-- tokens info table -->
+            <div class="panel panel-default">
+
+                <div class="panel-heading">
+                    <h3 class="panel-title">
+                        Credit cards
+                    </h3>
+                </div>
+
+                <table class="table table-hover">
+                    <thead>
+                        <tr>
+                            <th>
+                                Credit &amp; Debit Cards
+                            </th>
+                            <th><span ng-if="showTokenMethod">Method Name</span></th>
+                            <th class="text-center">
+                                Expires
+                            </th>
+                            <th></th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr ng-repeat="token in tokens">
+                            <td>{{token.Extra.type}} ending in {{token.Extra.number}}</td>
+                            <td><span ng-if="showTokenMethod">{{getPaymentMethodName(token.Desc)}}</span></td>
+                            <td class="text-center">{{parseDate(token.Extra.expiration_date)}}</td>
+                            <td class="text-right">
+                                <button class="btn btn-danger btn-block-xs"
+                                    ng-click="remove(token.ID)">
+                                    <i class="fa fa-remove fa-fw"></i>
+                                </button>
+                            </td>
+                        </tr>
+                    </tbody>
+                </table>
+
+                <div class="panel-footer">
+                    <button class="btn btn-primary"
+                            ng-click="popUpOpen()">
```

Comment:
@igorGalchenko disable this button when there are no enabled tokenable payment methods. You can use `ng-disabled`